usefulUtilities/scaleAvatar/scaleAvatar.js

```diff
@@ -0,0 +1,135 @@
+//
+//  scaleAvatar.js
+//
+//  Run script on client. In VR, when you hold down both grips and both triggers, 
+//  scale your avatar your avatar by moving your hands further away from eachother 
+//  to scale larger or closer together to scale smaller.
+//
+//  Created by Robin Wilson on 3/11/2019
+//  Copyright 2017 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+(function () {
+    var DEBUG = false;
+
+    // Returns boolean that all buttons are pressed
+    var isPressed = {
+        rightGrip: false,
+        leftGrip: false,
+        rightTrigger: false,
+        leftTrigger: false
+    };
+    function areAllPressed() {
+        return isPressed.rightGrip && isPressed.leftGrip && isPressed.rightTrigger && isPressed.leftTrigger;
+    }
+
+
+    // Sets up the scaling functionality if all buttons are pressed
+    var SCALE_BUFFER = 1, // increase/decrease the sensitivity of scaling
```

Comment:
if this variable is set to 1, do you even need to include it here and on line 55?